### DISCUSSION TITLE
Request: Add `usertime` option to `ckan.display_timezone`

### DISCUSSION THREAD

**Discussion by jqnatividad** [ckan/ckan#7248] [Ideas]
By setting `ckan.display_timezone` in the ckan.ini file, a CKAN instance can be configured to display a specified timezone, or the timezone of the server running CKAN.

https://docs.ckan.org/en/2.9/maintaining/configuration.html?highlight=timezone#ckan-display-timezone

It'd be nice if there's an additional option called `usertime` (so as not to confuse it with localtime) to display the timezone of the user browsing CKAN - i.e. if I'm browsing an instance in EST timezone, the datetimes are displayed using the EST timezone; if another user browsing the same instance is looking at the site in PST timezone, it displays date/times in the PST timezone.

**Comment by pdelboca** [comment]
Isn't this kind of logic handled usually at the frontend? 

We currently have https://github.com/ckan/ckan/blob/master/ckan/templates/snippets/local_friendly_datetime.html to try to sort this issue.

**Reply by jqnatividad** [reply]
Thanks for the tip @pdelboca !

Perhaps, that can be mentioned in https://docs.ckan.org/en/2.9/maintaining/configuration.html#ckan-display-timezone?
